src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -187,24 +204,23 @@ private Meal pickRandomMeal(List<Meal> mealList) {
         return (Meal)mealList.get(index);
     }
 
-    private Boolean isResultOfSearch(Meal meal, List<String> params) {
+    private int isResultOfSearch(Meal meal, List<String> params) {
+        int counter = 0;
         for (String param : params) {
-            if (meal.getTitle().contains(param)) {
-                return true;
-            } 
-            if (meal.getDescription().contains(param)) {
-                return true;
+            if (meal.getTitle().contains(param) ||
+                meal.getDescription().contains(param) ||
+                meal.getType().contains(param)) {
+                counter ++;
```

Comment:
I have never seen `++` operator with spacing between variable (it always was smth like `counter++`). Are you sure that it is right according to code style?

Reply:
I met both `i++` and `i ++`, but let first will be here.